graphics/zbar/Portfile

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               active_variants 1.1
+
+github.setup            mchehab zbar 0.23.90
+revision                0
```

Comment:
please add `github.tarball_from archive` underneath the `github.setup` line. Then clean the port and recalculate/update the checksums (`sudo port clean --all zbar; sudo port -dv checksum zbar`)